modules/auxiliary/admin/http/scadabr_credential_dump.rb

```diff
@@ -8,214 +8,242 @@ class MetasploitModule < Msf::Auxiliary
+          private_data: pass
+        })
       else
-        creds = { origin_type:     :service,
-                  module_fullname: fullname,
-                  private_type:    :nonreplayable_hash,
-                  private_data:    hash,
-                  username:        user }
+        creds.merge!({
+          private_type: :nonreplayable_hash,
+          private_data: hash
+        })
       end
 
-      creds.merge! service_data
-      credential_core = create_credential creds
-      login_data = { core: credential_core,
-                     access_level: (admin ? 'Admin' : 'User'),
-                     status: Metasploit::Model::Login::Status::UNTRIED }
-      login_data.merge! service_data
-      create_credential_login login_data
-    end
-
-    columns = 'Service', 'Host', 'Port', 'Username', 'Password'
-    service_cred_table = Rex::Text::Table.new 'Header'  => 'ScadaBR Service Credentials',
-                                              'Indent'  => 1,
-                                              'Columns' => columns
+      login_data = {
+        core: create_credential(creds),
+        access_level: (admin ? 'Admin' : 'User'),
+        status: Metasploit::Model::Login::Status::UNTRIED
+      }.merge(service_data)
 
-    system_settings = json['systemSettings'].first
-
-    unless system_settings['emailSmtpHost'].eql?('') || system_settings['emailSmtpUsername'].eql?('')
-      smtp_host = system_settings['emailSmtpHost']
-      smtp_port = system_settings['emailSmtpPort']
-      smtp_user = system_settings['emailSmtpUsername']
-      smtp_pass = system_settings['emailSmtpPassword']
-      vprint_good "Found SMTP credentials: #{smtp_user}:#{smtp_pass}@#{smtp_host}:#{smtp_port}"
-      service_cred_table << ['SMTP', smtp_host, smtp_port, smtp_user, smtp_pass]
+      create_credential_login(login_data)
     end
 
-    unless system_settings['httpClientProxyServer'].eql?('') || system_settings['httpClientProxyUsername'].eql?('')
-      proxy_host = system_settings['httpClientProxyServer']
-      proxy_port = system_settings['httpClientProxyPort']
-      proxy_user = system_settings['httpClientProxyUsername']
-      proxy_pass = system_settings['httpClientProxyPassword']
-      vprint_good "Found HTTP proxy credentials: #{proxy_user}:#{proxy_pass}@#{proxy_host}:#{proxy_port}"
-      service_cred_table << ['HTTP proxy', proxy_host, proxy_port, proxy_user, proxy_pass]
-    end
+    service_cred_table = Rex::Text::Table.new(
+      'Header' => 'ScadaBR Service Credentials',
+      'Indent' => 1,
+      'Columns' => ['Service', 'Host', 'Port', 'Username', 'Password']
+    )
 
     print_line
-    print_line user_cred_table.to_s
-    print_line
-    print_line service_cred_table.to_s
+    print_line(user_cred_table.to_s)
+
+    unless json['systemSettings'].nil?
+      system_settings = json['systemSettings'].first
+
+      unless system_settings['emailSmtpHost'].eql?('') || system_settings['emailSmtpUsername'].eql?('')
```

Comment:
Apparently, it's not called out by rubocop but we do try to follow the [Ruby Style Guide's](https://rubystyle.guide/#eql) recommendation on favoring `==` over `#eql?` when it is sufficient.

```suggestion
      unless system_settings['emailSmtpHost'] == '' || system_settings['emailSmtpUsername'] == ''
```

Reply:
I can update and force push if you like. Else go for it.

This is existing code. This module is pretty old. I changed a few `.eql?` but evidently missed a couple.